stake-pool/cli/src/main.rs

```diff
@@ -1611,6 +1856,61 @@ fn main() {
                           Defaults to the token-owner's associated pool token account. \
                           Creates the account if it does not exist."),
             )
+            .arg(
+                Arg::with_name("referrer")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .help("Account to receive the referral fees for deposits. \
+                          Defaults to the pool manager."),
```

Comment:
```suggestion
                          Defaults to the token receiver."),
```

Reply:
I believe the referrer is supposed to default to the pool manager if none is provided